utils/utils-lib/src/tracing_telemetry.rs

```diff
@@ -155,6 +155,34 @@ pub fn init_tracing_ext<T: std::net::ToSocketAddrs>(
     };
 }
 
+/// Init tracing for utility binary, ex csi-node for fsfreeze.
+pub fn init_tracing_for_utility_binary(stderr: bool) {
```

Comment:
Let's create a Builder type instead:
```rust
struct FmtStyle { Compact, Pretty };
struct TracingTel {
  writer: FmtLayer,
  style: FmtStyle,
  colours: bool
}
```
 allowing the code in the csi to be written like so:
```rust
TracingTel::builder().with_writer(FmtLayer::Stdout).with_style(FmtStyle::Compact).with_colours(false)
```